### DISCUSSION TITLE
Talos metal iso and enable dhcp6

### DISCUSSION THREAD

**Discussion by torgiren** [siderolabs/talos#8645] [Q&A]
Hi,
I cannot find the answer neither in docs or google...

I'm trying to run metal iso on libvirt, but I want it to use ipv6.
I've configured dhcp to serve dhcpv6, but talos but default uses only dhcp4
I know that I could use talos.config=metal-iso to reconfigure that, but that also make the installation

I'm looking for some way, (maybe talos factory options) to make the iso image that will have dhcp6 enabled by default, so I can run talosctl apply-configuration etc using ipv6 addresses

**Comment by torgiren** [comment]
or maybe there is a way to build an iso with my custom network configuration?

**Reply by smira** [reply]
I don't think Talos has a way to enable DHCPv6 other than via a machine configuration.

Your options are probably limited to IPv6 SLAAC, or using temporary IPv4 DHCP address to apply initial config and then disable DHCPv4.

**Reply by torgiren** [reply]
ah, the problem with ipv4 is that this is working on libvirt, co ipv4 is behind the nat,
and ipv6 would be public...
but, if there is no other way, I will thik how to setup a proxy or something
